Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -130,6 +140,30 @@ final class PledgePaymentPlanOptionView: UIView {
     ])
 
     self.termsOfUseButton.setContentHuggingPriority(.required, for: .horizontal)
+
+    self.ineligibleBadgeView.addSubview(self.ineligibleBadgeLabel)
```

Comment:
Would be slightly cleaner/more brief if the `ineligibleBadgeView` was its own UIView subclass

Reply:
Added to the follow up/clean up ticket https://kickstarter.atlassian.net/browse/MBL-1940